src/serviceconnector-passwordless/azext_serviceconnector_passwordless/_credential_free.py

```diff
@@ -162,6 +162,29 @@ def getTargetHandler(cmd, target_id, target_type, auth_info, client_type, connec
 
 
 class TargetHandler:
+    cmd = None
```

Comment:
Please define them in the _init_ function instead of class property

Reply:
updated